Gems/Atom/Component/DebugCamera/Code/Include/Atom/Component/DebugCamera/CameraComponent.h

```diff
@@ -102,25 +104,41 @@ namespace AZ
             void SetFrustumHeight(float height) override;
             void SetOrthographic(bool orthographic) override;
             void SetOrthographicHalfWidth(float halfWidth) override;
+            void SetXRViewQuaternion(const AZ::Quaternion& viewQuat, uint32_t xrViewIndex) override;
+
             void MakeActiveView() override;
             bool IsActiveView() override;
             AZ::Vector3 ScreenToWorld(const AZ::Vector2& screenPosition, float depth) override;
             AZ::Vector3 ScreenNdcToWorld(const AZ::Vector2& screenPosition, float depth) override;
             AZ::Vector2 WorldToScreen(const AZ::Vector3& worldPosition) override;
             AZ::Vector2 WorldToScreenNdc(const AZ::Vector3& worldPosition) override;
-
+            
             // RPI::WindowContextNotificationBus overrides...
             void OnViewportResized(uint32_t width, uint32_t height) override;
 
             void UpdateAspectRatio();
-
             void UpdateViewToClipMatrix();
 
-            RPI::ViewPtr m_view = nullptr;
-            
+            //! Enum to describe view type
+            enum class ViewType : uint32_t
+            {
+                Default = 0,
+                XrLeft,
+                XrRight,
+                Count
+            };
+            AZStd::vector<RPI::ViewPtr> m_cameraViews;
```

Comment:
Can you add some comment for the camera views array? 

It's a bit confusing if the size of the vector is the same as the ViewType count.